projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -106,6 +106,12 @@ public JobImageBuilder(
     this.kubernetesResources = kubernetesResources;
   }
 
+  /** see {@link JobImageBuilder#buildImage(String, DataJob, JobDeployment, Boolean, String)}} */
+  public boolean buildImage(
+      String imageName, DataJob dataJob, JobDeployment jobDeployment, Boolean sendNotification)
+      throws ApiException, IOException, InterruptedException {
+    return buildImage(imageName, dataJob, jobDeployment, sendNotification, null);
```

Comment:
nit: I like a style where parameters that are meant to be optional (could be null/empty by design), are actually wrapped with Optional